tricycle_steering_controller/test/test_tricycle_steering_controller.cpp

```diff
@@ -75,7 +75,7 @@ TEST_F(TricycleSteeringControllerTest, check_exported_intefaces)
     state_intefaces.names[STATE_STEER_AXIS],
     controller_->front_wheels_state_names_[0] + "/" + steering_interface_name_);
 
-  // check ref itfs
+  // check ref itfsTIME
```

Comment:
Typo?

Reply:
Locally it's showing that the change to this file has been applied but there is no commit for this change.

Since the change is already applied locally, I can't set the line to be `// check ref itfsTIME` and then commit it since there is no change from what it already is.

I also can't find a 'commit' button not on github nor in my IDE to accept the suggested change for this specific file, the other 3 files have been adjusted.

any thoughts?